apps-rendering/src/components/headline.tsx

```diff
@@ -153,12 +177,57 @@ const getStyles = (format: ArticleFormat): SerializedStyles => {
 	}
 };
 
-const Headline = ({ item }: Props): ReactElement => (
-	<h1 css={getStyles(item)}>
-		<span>{item.headline}</span>
-		<StarRating item={item} />
-	</h1>
-);
+const headlineStyles = (format: ArticleFormat): SerializedStyles => {
+	return css`
+		color: ${text.headline(format)};
+		background-color: ${neutral[0]};
+		position: relative;
+		white-space: pre-wrap;
+		padding: 0 ${remSpace[1]};
+		box-decoration-break: clone;
```

Comment:
Actually, scrape that - it is needed to maintain the left and right padding if the headline spans across multiple lines. Otherwise the padding will be applied to the first line only.

| With `box-decoration-break: clone` | Without `box-decoration-break: clone` |
| ---- | ---- |
| <img width="388" alt="image" src="https://user-images.githubusercontent.com/57295823/163371876-3e61a3b0-11bf-4c3f-b7d9-22bf2ee1bad9.png"> |  <img width="386" alt="image" src="https://user-images.githubusercontent.com/57295823/163371935-06f66c33-e23b-4ba1-890b-011704dda825.png"> |